src/hooks/manual_block_list.ts

```diff
@@ -0,0 +1,1197 @@
+  '0x7cc18b22e0034b5a084498385e9ea38161a98395': {},
+  '0x6388ca4f347c88ea6b8cd64575e08d6fd4cda418': {},
+  '0xf215173e11af103034234f375c2c9bb5d9f5df67': {},
+  '0xa499d18f5e2a54cdc8ed00dfe88416e3eb6bc5df': {},
+  '0x7e4f81b93480e87153c5a888abbb26fdca371ef7': {},
+  '0x01e1740b500ccde265e2ed5ebf0fd7c56cead04d': {},
+  '0x27f278045b3bb84501a3d28f60e30ff29f5140af': {},
+  '0x0452102f49457cc55c80456de2638e1fbfda336b': {},
+  '0x1a6334b9463d27f71ffb04cf8bafe9148c8b0249': {},
+  '0x48be9c7eb6b60e148165ab9a9d3334aa1fbae7a2': {},
+  '0x7ea1682d542e3583ff43697b72b7a1ad044913ff': {},
+  '0xcd10666b67765a438747b5551d0646393df7b215': {},
+  '0x9694188c0df0041ad1c8b9034bb47b9261a6b6aa': {},
+  '0x43df3b2d416964840a9fc8ddb9cd2607c5137a5b': {},
+  '0x05b69e2f47ac8d4f87d46c478c0e10c1c42809e8': {},
+  '0x80b8c2257f75ce3df03cb35a63351bd3137026f5': {},
+  '0x488bbad97a096f9fee791bd0dac7f8e4116722aa': {},
+  '0x48f570d888b6705c446cc34a3e4cc2c9abb9d58d': {},
+  '0x301ad935653f636802797bf50863f3cc5ced2e04': {},
+  '0x9b689e504711b0b9794772996bcd0a28cd46d2ff': {},
+  '0x0385d54a716bf07f20d025fd828dec866090c70c': {},
+  '0x439bcb76332abcde6e2ee47b0f62d198788c48d7': {},
+  '0x7f4330edf6276f6950f7d5f4040a845faf1177c6': {},
+  '0xca31464f458f8373975a97598606b299a97b52d4': {},
+  '0x0e8f4ba30c65478fd52ebe619b554728b8cd02a1': {},
+  '0x8e73d45ef12b05f2c1b55e31ee848bfc10095b48': {},
+  '0x0e6949212313b0af8b5645c43509f45a3171d35b': {},
+  '0x250f1a1f2a2ad9f90adb94fd46fb8767e6efb11b': {},
+  '0x5482a8af72ac9a8ea78ff95969c6768b69de33cf': {},
+  '0xbc7b57f1f234f2d4371ad048687f8de6f89c3cdb': {},
+  '0x8763eadfa53e23db514b9f7b90e523f1185dcbef': {},
+  '0xbe4b124b87da0cf72d2846ac0eca11d6f7a1812b': {},
+  '0xb6fe09fd7890815b77dae0ac49de3f8062c3e471': {},
+  '0xa54202eab3235411f987eda8bac397f666971dad': {},
+  '0x71c8757680293e891636afc9f7aaee1f98f6986b': {},
+  '0x9b2ce22e91b6f4b62f31f47fe10fec7b76fc9dcf': {},
+  '0xd8a53aefae76855a97ac0a2c8050d615862a543b': {},
+  '0x1a185641d874d0194e89610fc19ab015aba3ab72': {},
+  '0x6e6dbf866d189ebaca14e638400106f9340ebcbc': {},
+  '0xb092b630291568e8c95fcc2e6e31fd1df4fc2695': {},
+  '0xe20c7b990b7306ef2b656c7c31b4f1d45981a10a': {},
+  '0xfed336b52a60e28e99604af6136a846a0402711d': {},
+  '0x6e49365a9318a0e868e9257476c24afe94709486': {},
+  '0x6bcb3061a6ad777a14cadd4f93ecef211bcf22af': {},
+  '0xf07d5a81c4d85e4f28d9314380b1d68e16cf21bc': {},
+  '0x270279faa3421d3e86a2ed72f1826fd55dae3314': {},
+  '0x2cc061427ad01116d2d57d3a233efd326c0175ff': {},
+  '0x4447bcbe4be076c1a188a0178c5137da218698cd': {},
+  '0x8b23eebeefb34537c94a14430c564b3827275823': {},
+  '0xfdcaf6fed2e91a2f7792d488820de7fdc590a6d5': {},
+  '0x4aa329613c50d5ee76e8c3bb55185dc357e4978f': {},
+  '0x8abd9c7efa840907a44f633dd91ce4137600308b': {},
+  '0x97fe4ed781ba0a07fbd9323109d0458478aced9e': {},
+  '0xed7f81c87bc15b4be85e26cb49560528838dbd58': {},
+  '0x4deeb19cc5be551e3e7ca1d54e7960542ceed642': {},
+  '0xbe437804b82995c0c9ed63e59fe2d090f312f2b6': {},
```

Comment:
Why not just make this a set?

Reply:
cause faster to paste in what i have

Reply:
same end result so working faster not smarter rn